.codeclimate.yml

```diff
@@ -0,0 +1,19 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+  eslint:
+    enabled: true
+    channel: "eslint-3"
+ratings:
+  paths:
+  - "**.js"
+exclude_paths:
+- dist/
+- node_modules/
+- test/
```

Comment:
I'd like to make sure linting runs against the test suite as well. Is there a specific reason why we want to blacklist the tests here @sivakumar-kailasam?